src/routes/(inner)/examples/context-menus/+page.svelte

```diff
@@ -1,7 +1,13 @@
+<script>
+	import App from '$lib/hooks/useClick/App.test.svelte';
+</script>
+
 <div class="space-y-10">
 	<!-- Header -->
 	<header class="card card-gradient space-y-8">
 		<h1 class="h1"><span>Context Menus</span></h1>
 		<p>Coming Soon!</p>
 	</header>
 </div>
+
+<App enableHover={true} />
```

Comment:
This looks like some testing features that go left on the page. Going forward I might suggest creating a `/sandbox` route that we don't link to in the navigation. Just a "hidden" test area for dev work like this.